package.json

```diff
@@ -18,17 +18,19 @@
     "test:unit:browserstack:local": "karma start karma.browserstack.conf.js",
     "test:unit:browserstack:ci": "karma start karma.browserstack.conf.js --browsers bs_chrome_windows,bs_safari_macos,bs_firefox_windows,bs_ie_windows --single-run --reporters spec,BrowserStack",
     "test:unit:coverage": "karma start --single-run --reporters spec,coverage",
-    "test:functional": "rollup -c --environment BASE_CODE,NPM_PACKAGE_LOCAL && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
+    "test:functional": "npm run test:functional:build:int && EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
     "test:functional:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" ./scripts/watchFunctionalTests.js",
-    "test:functional:ci:build": "rollup -c --environment BASE_CODE,NPM_PACKAGE_LOCAL",
+    "test:functional:build:int": "rollup -c --environment BASE_CODE_MIN,STANDALONE,NPM_PACKAGE_LOCAL",
+    "test:functional:build:prod": "rollup -c --environment BASE_CODE_MIN,NPM_PACKAGE_PROD",
     "test:functional:ci:predeploy": "ALLOY_ENV=\"int\" testcafe 'browserstack:chrome@87:Windows 10','browserstack:ie@11:Windows 10','browserstack:firefox@85:Windows 10','browserstack:safari@14.0:OS X Big Sur'",
     "test:functional:ci:dev": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe 'browserstack:chrome@87:Windows 10','browserstack:ie@11:Windows 10','browserstack:firefox@85:Windows 10','browserstack:safari@14.0:OS X Big Sur'",
     "test:functional:ci:prod": "ALLOY_ENV=\"prod\" testcafe 'browserstack:chrome@87:Windows 10','browserstack:ie@11:Windows 10','browserstack:firefox@85:Windows 10','browserstack:safari@14.0:OS X Big Sur'",
     "test:functional:ci:edgepredeploy": "ALLOY_ENV=\"prod\" EDGE_BASE_PATH=\"ee-pre-prd\" testcafe 'browserstack:chrome@87:Windows 10','browserstack:ie@11:Windows 10','browserstack:firefox@84:Windows 10','browserstack:safari@14.0:OS X Big Sur'",
     "test:functional:api": "testcafe chrome:headless",
     "sandbox:build": "cd sandbox && npm run build",
     "dev": "concurrently --names build,sandbox \"rollup -c -w --environment SANDBOX\" \"cd sandbox && export REACT_APP_NONCE=321 && npm start\"",
```

Comment:
It seems like we shouldn't have to build the standalone variant (which is done through the `build` npm script) and the sandbox variant (which is done on this line). I'm kind of losing track of things though.

Reply:
I talked to @jonsnyder about this. I got confused with how concurrently works. My comment can be ignored.

Reply:
It isn't calling build. `--names build,sandbox` is just naming the two commands for logging.